xml/System.Reflection.Metadata/ExceptionRegion.xml

```diff
@@ -49,7 +49,7 @@
         <ReturnType>System.Reflection.Metadata.EntityHandle</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Returns a TypeRef, TypeDef, or TypeSpec handle if the region represents a catch, nil token otherwise (<see langword="default" /> (<see cref="T:System.Reflection.Metadata.EntityHandle" />)).</summary>
```

Comment:
```suggestion
        <summary>Gets a TypeRef, TypeDef, or TypeSpec handle if the region represents a catch, or a nil token otherwise (<see langword="default" />(<see cref="T:System.Reflection.Metadata.EntityHandle" />)).</summary>
```